README.md

```diff
@@ -189,6 +189,11 @@ $ conda activate <CONDA_NAME>
 $ call "C:\Program Files\Microsoft Visual Studio\<VERSION>\Community\VC\Auxiliary\Build\vcvarsall.bat" x64
 ```
 
+A conda environment is not required.  You can also do a PyTorch build in a
+standard virtual environment, e.g., created with tools like `uv`, provided
+your system has all the necessary dependencies (e.g., cmake, ninja, MKL)
```

Comment:
```suggestion
your system has all the necessary dependencies unavailable as pip packages (e.g., CUDA, MKL)
```

cmake and ninja will be installed as part of requirements.txt with the recent enough version

Reply:
We install mkl thru pypi as well, but also this is so x86-like :)